features/pentaho-features/pentaho-karaf-features-standard/src/main/feature/feature.xml

```diff
@@ -232,7 +232,7 @@
   </feature>
 
   <!-- Separate feature for jackson-* bundles in scope of BACKLOG-20783 -->
-  <feature name="pentaho-jackson" description="Jackson 2.9.x support" version="1.0">
+  <feature name="pentaho-jackson" description="Jackson 2.10.x support" version="1.0">
```

Comment:
We should start versioning the feature as well. It's been stuck on version 1.0 forever despite having changed. 
Using the fasterxml-jackson version seems appropriate.